### PR TITLE
[15.0][IMP] base_import_pdf_by_template: Add fixed value compatibility to different types of fields

### PR DESCRIPTION
**Pull request by victoralmau**
Add fixed value compatibility to different types of fields

Please @pedrobaeza and @pilarvargas-tecnativa can you review it?

@Tecnativa TT51379

### CODE REVIEW

File: base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py [hunk 1]
```diff
@@ -122,6 +122,19 @@ def action_process(self):
         self.attachment_id.write({"res_model": record._name, "res_id": record.id})
         return record
 
+    def _add_log_error_text(self, field_name, value):
```
Comment (pedrobaeza) [inline base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py:125]: This change seems unrelated

Comment (victoralmau) [reply base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py:125]: This change is to avoid adding code that repeats things, similar to what is suggested at https://github.com/OCA/edi/pull/1051#discussion_r1805914480.

Comment (pedrobaeza) [reply base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py:125]: Yeah, but it's unrelated to the other change, so please put it in a different commit.

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
Ping @pilarvargas-tecnativa

**Review by pilarvargas-tecnativa — APPROVED** [review]
